app/api/admin/contact-submissions/analytics/route.ts

```
@@ -0,0 +1,389 @@
+      const date = submission.createdAt;
+      let dateKey: string;
+
+      switch (getChartGrouping(period)) {
+        case "hour":
+          dateKey = date.toISOString().substring(0, 13) + ":00";
+          break;
+        case "day":
+          dateKey = date.toISOString().substring(0, 10);
+          break;
+        case "week":
+          const weekStart = new Date(date);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)